src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -1629,14 +1668,26 @@ void addToCatalog(ImportTask task) throws IOException {
         // changed to deal with name clashes
         // resource.setNativeName(name);
         resource.setEnabled(true);
+        LOGGER.log(
+                Level.FINE,
+                "Creating target resource {0} for task {0}",
+                new Object[] {resource, task.getId()});
         catalog.add(resource);
 
         // add the layer (and style)
         if (layer.getDefaultStyle().getId() == null) {
+            LOGGER.log(
+                    Level.FINE,
+                    "Creating default style {0} for task {0}",
+                    new Object[] {layer.getDefaultStyle(), task.getId()});
             catalog.add(layer.getDefaultStyle());
         }
 
         layer.setEnabled(true);
+        LOGGER.log(
+                Level.FINE,
+                "Creating target layer  {0} for task {0}",
```

Comment:
Second parameter should be {1} to get the task id
Also, I see 2 whitespaces between layer and {0}. Not a critical problem :) but since you need to fix the text using {1} I would also remove 1 whitespace

Reply:
Good catch, fixed.